x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/details_page/components/package_policies/no_package_policies.tsx

```diff
@@ -14,7 +14,9 @@ import { pagePathGetters, INTEGRATIONS_PLUGIN_ID } from '../../../../../constant
 
 export const NoPackagePolicies = memo<{ policyId: string }>(({ policyId }) => {
   const { application } = useStartServices();
-  const canWriteIntegrationPolicies = useAuthz().integrations.writeIntegrationPolicies;
+  const authz = useAuthz();
+  const canWriteIntegrationPolicies =
```

Comment:
it could be worth covering non-trivial conditions with unit tests

Reply:
+1 actually I think it will make sense to change the authz.integrations.writeIntegrationPolicies to require `authz.fleet.allAgentPolicies` I will make that change and it should simplify that condition